src/routes/ViewAllLogs/ViewAllLogs.test.js

```diff
@@ -0,0 +1,243 @@
+import React from 'react';
+
+import {
+  fireEvent,
+  render,
+} from '@testing-library/react';
+
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import { noop } from 'lodash';
+
+import {
+  buildMutator,
+  Harness,
+} from '@folio/stripes-data-transfer-components/test/helpers';
+
+import '../../../test/jest/__mock__';
+
+import { translationsProperties } from '../../../test/jest/helpers';
+
+import ViewAllLogs from './ViewAllLogs';
+
+const mutator = buildMutator({
+  initializedFilterConfig: {
+    update: noop,
+    replace: noop,
+  },
+  query: {
+    replace: noop,
+    update: noop,
+  },
+  records: {
+    DELETE: noop,
+    POST: noop,
+    PUT: noop,
+    cancel: noop,
+  },
+  resultOffset: { replace: noop },
+  resultCount: {
+    replace: noop,
+    update: noop,
+  },
+});
+
+const resources = {
```

Comment:
You can use `buildResources` helper from the `stripes-data-transfer-components` to build this